miner/src/pool/mod.rs

```diff
@@ -175,3 +182,19 @@ impl txpool::VerifiedTransaction for VerifiedTransaction {
 		&self.sender
 	}
 }
+
+impl ScoredTransaction for VerifiedTransaction {
+	fn priority(&self) -> Priority {
+		self.priority
+	}
+
+	/// Gets transaction gas price.
+	fn gas_price(&self) -> U256 {
```

Comment:
Might be better to return `&` here, to avoid copying.

Reply:
Reworked